torch/ao/quantization/quantize_pt2e.py

```diff
@@ -78,6 +109,10 @@ def convert_pt2e(
 
     pm = PassManager([PortNodeMetaForQDQ()])
     model = pm(model).graph_module
+
+    if fold_quantized_weight:
```

Comment:
Is it a typo `fold_quantized_weight`? Looks like the input parameter is `fold_quantize`.

Reply:
oh right, I'll update this soon